vscode-dotnet-runtime-library/src/Utils/FileUtilities.ts

```diff
@@ -0,0 +1,65 @@
+ /* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+ import * as eol from 'eol';
+ import * as fs from 'fs';
+ import * as path from 'path';
+ import * as os from 'os';
+ import * as proc from 'child_process';
+/* tslint:disable:no-empty */ // allow empty constructor
+
+export class FileUtilities {
+    constructor()
+    {
+    }
+
+    public static writeFileOntoDisk(scriptContent: string, filePath: string)
```

Comment:
Same static/ seam comment here as on the version resolver. It might actually be somewhat more important here, since if we're able to substitute out this class when testing then that would mean we can avoid touching the file system.